pkg/kyverno/common/fetch.go

```diff
@@ -272,6 +272,33 @@ func convertResourceToUnstructured(resourceYaml []byte) (*unstructured.Unstructu
 	return resource, nil
 }
 
+// GetPatchedResource converts raw bytes to unstructured object
+func GetPatchedResource(patchResourceBytes []byte) (patchedResource unstructured.Unstructured, err error) {
+	var getErrString string
+	files, splitDocError := utils.SplitYAMLDocuments(patchResourceBytes)
```

Comment:
can we just call `splitDocError` as `err` ?

Reply:
Since in the function definition also, you have defined `err` for error
```
func GetPatchedResource(patchResourceBytes []byte) (patchedResource unstructured.Unstructured, err error)
```

Reply:
ok, np. Replaced `splitDocError` by `err`